packages/gatsby/content/features/zero-installs.md

```diff
@@ -58,3 +58,9 @@ $> yarn install --check-cache
 ```
 
 This way Yarn will re-download the package files from whatever their remote location would be and will report any mismatching checksum.
+
+### What does this do to my repository size?
+
+Every time you update a dependency and commit this, the repository will grow.
```

Comment:
likely grow?
```suggestion
Every time you update a dependency and commit this, the repository will likely grow.
```

Although not common, some popular projects do prioritize reducing install sizes.

Reply:
Even if the install size is smaller, the repo itself will grow, as the previous version will be part of the history. So it will always grow, as far as I understand.

Reply:
But that does mean that I should clarify this a bit more.

Reply:
I've added some clarification in the sentence